src/block-management/inline-toolbar/index.js

```diff
@@ -56,27 +56,30 @@ export default class InlineToolbar extends React.Component<PropsType> {
 			<View style={ styles.toolbar } >
 				<ToolbarButton
 					accessibilityLabel={ __( 'Move up' ) }
-					label={ __( 'Move up' ) }
+					title={ __( 'Move up' ) }
 					isDisabled={ ! this.props.canMoveUp }
 					onClick={ this.onUpPressed }
 					icon="arrow-up-alt"
+					extraProps={ { hint: __( 'Double tap to move up the block' ) } }
```

Comment:
> 'Double tap to move up the block'

I think that "Double tap to move **the block up**" would be more correct, but I'm not sure if the original is wrong or just another valid way to say it.

We could check with some native English speaker :)

Reply:
maybe @mattmiklic can help us on this :)

Reply:
The phrase "Double tap to move up the block" is technically correct, but may be ambiguous, because, in that case, it is not clear whether "the block" is the direct object, or the object of the preposition (i.e. the thing being moved, or where we are moving to, respectively).

With the phrase "Double tap to move the block up", it is more clear that "the block" is the thing that we will move.

Reply:
Cool, thanks for the explanation @mkevins ! 🎉 Will push an update

Reply:
FWIW. In the PR @etoledom mentioned I included a row number in the hint. I'm thinking this will have a positive impact on clearing up any ambiguity(?).